public/css/icinga/menu.less

```diff
@@ -278,12 +280,12 @@
       border-left: 1px solid @gray-light;
       content: "";
       display: block;
-      height: 1.1em;
```

Comment:
These 1.1em were referenced in previous changes, as seen in line 294. I think, now is the time to define a variable for this.